starter-code/client/components/NYCMap.js

```diff
@@ -0,0 +1,183 @@
+    },
+    {
+        "featureType": "poi.business",
+        "elementType": "all",
+        "stylers": [
+            {
+                "visibility": "off"
+            }
+        ]
+    },
+    {
+        "featureType": "poi.medical",
+        "elementType": "geometry",
+        "stylers": [
+            {
+                "color": "#fbd3da"
+            }
+        ]
+    },
+    {
+        "featureType": "poi.park",
+        "elementType": "geometry",
+        "stylers": [
+            {
+                "color": "#bde6ab"
+            }
+        ]
+    },
+    {
+        "featureType": "road",
+        "elementType": "geometry.stroke",
+        "stylers": [
+            {
+                "visibility": "off"
+            }
+        ]
+    },
+    {
+        "featureType": "road",
+        "elementType": "labels",
+        "stylers": [
+            {
+                "visibility": "off"
+            }
+        ]
+    },
+    {
+        "featureType": "road.highway",
+        "elementType": "geometry.fill",
+        "stylers": [
+            {
+                "color": "#ffe15f"
+            }
+        ]
+    },
+    {
+        "featureType": "road.highway",
+        "elementType": "geometry.stroke",
+        "stylers": [
+            {
+                "color": "#efd151"
+            }
+        ]
+    },
+    {
+        "featureType": "road.arterial",
+        "elementType": "geometry.fill",
+        "stylers": [
+            {
+                "color": "#ffffff"
+            }
+        ]
+    },
+    {
+        "featureType": "road.local",
+        "elementType": "geometry.fill",
+        "stylers": [
+            {
+                "color": "black"
+            }
+        ]
+    },
+    {
+        "featureType": "transit.station.airport",
+        "elementType": "geometry.fill",
+        "stylers": [
+            {
+                "color": "#cfb2db"
+            }
+        ]
+    },
+    {
+        "featureType": "water",
+        "elementType": "geometry",
+        "stylers": [
+            {
+                "color": "#a2daf2"
+            }
+        ]
+    }
+]
+
+export default function NYCMap() {
+    const position = { lat: 40.712775, lng: -74.005973 };
+
+    return(
+        <APIProvider apiKey={process.env.NEXT_PUBLIC_GOOGLE_MAPS_API_KEY}>
+            <div style={{ height: "90vh"}}>
+            <Map 
+                zoom={13} 
+                center={position} 
+                options={{
+                styles: mapStyles, 
+                draggable: true, 
+                zoomControl: true
+                }}
+            />
+            </div>
+        </APIProvider>
+    )
+}
+
+
+
+/*
+export default function NYCMap() {
+    const position = { lat: 40.7128, lng: -74.0060 };
+
+    return(
+        <APIProvider apiKey="AIzaSyBMpZf4jDALGdvq2QYbNUl3ZuquMaK7I_k">
+            <div style={{ height: "100vh"}}>
+                <Map zoom={13} center={position}></Map>
+            </div>
+        </APIProvider>
+    )
+}
+*/
```

Comment:
Hey, can we remove this commented out code? It looks like the api key is still visible.

Reply:
Sounds good!